pkg/util/crypt/crypt_dev.go

```diff
@@ -71,29 +70,22 @@ func (crypt *Device) CloseCryptDevice(path string) error {
 // Keys should be non-interactive, preferably in a keyfile that can
 // be passed to cryptsetup utility
 func (crypt *Device) ReadKeyFromStdin(confirm bool) (string, error) {
-
-	fmt.Print("Enter the Key: ")
-	password, err := terminal.ReadPassword(int(syscall.Stdin))
+	pass1, err := sypgp.AskQuestionNoEcho("Enter a passphrase: ")
 	if err != nil {
-		sylog.Fatalf("Error parsing the key: %s", err)
+		return "", fmt.Errorf("unable getting input: %s", err)
 	}
 
-	input := string(password)
-	fmt.Println()
 	if confirm {
-		fmt.Print("Confirm the Key: ")
-		password2, err := terminal.ReadPassword(int(syscall.Stdin))
+		pass2, err := sypgp.AskQuestionNoEcho("Confirm the passphrase: ")
 		if err != nil {
-			sylog.Fatalf("Error parsing the key: %s", err)
+			return "", fmt.Errorf("unable parsing input: %s", err)
 		}
-		input2 := string(password2)
-		fmt.Println()
-		if input != input2 {
-			return "", errors.New("Keys don't match")
+		if pass1 != pass2 {
```

Comment:
using variable names like pass1, pass2, pass3 are not a good practice. Better to have in this case a "passphrase" instead of "pass1" variable and then a "confirmPassphrase" variable instead of "pass2" to avoid the added consecutive numbers..

Reply:
Thanks :+1: fixed